### ISSUE TITLE
Example GPS does not work on newer versions android

### ISSUE THREAD

**Issue by edsonitk** [kivy/plyer#505]
In the most recent versions of the android, the coordinates are not updated by running the GPS example. No error occurs.

**Comment by justinb4003** [comment]
I'm having the same difficulty. The stock examples do not ever hit the on_status or on_location functions.  I"m deploy to Android 9 (Build PQ3A.190705.001) on a Google Pixel (Verizon).

**Comment by foobarbecue** [comment]
Ugh... plyer android GPS has been broken for 3 years?

**Comment by foobarbecue** [comment]
My error is:
```
06-04 13:52:27.210  9433  9433 I python  : ===== Python/java method missing ======
06-04 13:52:27.210  9433  9433 I python  : Python class:<plyer.platforms.android.gps._LocationListener object at 0x6f6b8c3700>
06-04 13:52:27.210  9433  9433 I python  : Java method name:onLocationChanged
06-04 13:52:27.210  9433  9433 I python  : Signature: (Ljava/util/List;)V
06-04 13:52:27.210  9433  9433 I python  : =======================================
06-04 13:52:27.210  9433  9433 I python  : 
06-04 13:52:27.212  9433  9433 I python  :  Traceback (most recent call last):
06-04 13:52:27.212  9433  9433 I python  :    File "jnius/jnius_proxy.pxi", line 50, in jnius.jnius.PythonJavaClass.invoke
06-04 13:52:27.213  9433  9433 I python  :    File "jnius/jnius_proxy.pxi", line 74, in jnius.jnius.PythonJavaClass._invoke
06-04 13:52:27.213  9433  9433 I python  :  NotImplementedError: The method ('onLocationChanged', ('V', ('Ljava/util/List;',))) is not implemented
```

**Comment by foobarbecue** [comment]
https://groups.google.com/g/kivy-users/c/S2TpsuKUpxk

**Comment by foobarbecue** [comment]
probably the same as https://github.com/kivy/plyer/issues/657

**Comment by tito** [comment]
Fixed in #665
